_includes/related-list-item.html

```diff
@@ -1,5 +1,5 @@
   <div class="post">
-  <div class="post_img" style="background-image:url({{ site.url }}{{ site.baseurl }}assets/images/blogcover/mini/{{ post.cover }});"></div>
+  <div class="post_img" style='background-image:url({% asset blogcover/"{{ post.cover }}" srcset:width="1300 2x" srcset:width="650 1x" @path %});'></div>

```

Comment:
That doesn't look right. Surely these images should not be 650 pixels, wide, right?

Reply:
I used the settings as for blog images. What parameters do you suggest to use?

Reply:
@vpetersson What parameters do you suggest to use? Since these are background images for blocks, they are stretched to fit the size of the block. And the block size depends on the width and height of the user's browser window.

Reply:
I don't know enough about the implementation, but perhaps [this](https://stackoverflow.com/questions/21154631/setting-background-image-with-jekyll-assets) will help. [This SO post](https://stackoverflow.com/questions/16154494/retina-displays-high-res-background-images) might also help further.